src/internal/components/cartesian-chart/left-labels.tsx

```diff
@@ -87,14 +90,25 @@ function LeftLabels({
                   className={clsx(styles.grid, styles.ticks_line)}
                   x1={-TICK_LENGTH}
                   y1={0}
-                  x2={width}
+                  x2={plotWidth}
                   y2={0}
                   aria-hidden="true"
                 />
               )}
-              <text className={styles.ticks__text} x={-(TICK_LENGTH + TICK_MARGIN)} y={0}>
-                {lines.join(' ')}
-              </text>
+
+              {lines.map((line, lineIndex) => {
```

Comment:
Turns out, the left cartesian chart labels never were multiline although the collision detection is supported for that case. Made it multiline to make the truncation target individual lines same as in the pie chart. It is now also possible to control the wrapping on the consumer end.